.github/workflows/docker-release.yml

```diff
@@ -40,13 +41,22 @@ jobs:
         with:
           username: "${{ secrets.DOCKERHUB_USER }}"
           password: "${{ secrets.DOCKERHUB_SECRET }}"
+      - name: Login to Quay
+        uses: docker/login-action@v2
+        with:
+          registry: "${{ env.QUAY }}"
+          username: "${{ secrets.QUAY_USER }}"
+          password: "${{ secrets.QUAY_TOKEN }}"
       - name: Build and push container to Dockerhub
         uses: docker/build-push-action@v4
         with:
           context: .
           platforms: ${{ env.PLATFORMS }}
           tags: ${{ env.IMAGE_REPOSITORY }}:latest ,
             ${{ env.IMAGE_REPOSITORY }}:${{ env.LATEST_VERSION }} ,
-            ${{ env.IMAGE_REPOSITORY }}:${{ env.LATEST_VERSION }}-${{ env.DATE }}
+            ${{ env.IMAGE_REPOSITORY }}:${{ env.LATEST_VERSION }}-${{ env.DATE }} ,
+            ${{ env.QUAY }}/${{ env.IMAGE_REPOSITORY }}:latest ,
+            ${{ env.QUAY }}/${{ env.IMAGE_REPOSITORY }}:${{ env.LATEST_VERSION }} ,
+            ${{ env.QUAY }}/${{ env.IMAGE_REPOSITORY }}:${{ env.LATEST_VERSION }}-${{ env.DATE }}
```

Comment:
could you make pushing to quay a separate step to pushing to dockerhub?

Reply:
Yes, one possibility would be [this](https://docs.docker.com/build/ci/github-actions/copy-image-registries/). Not sure if this is cleaner though.

Alternatively I could add a second build and push step, which would require rebuilding the image a second time if we are okay with that.

Reply:
> Yes, one possibility would be [this](https://docs.docker.com/build/ci/github-actions/copy-image-registries/).
This approach looks good to me

Reply:
Updated